pkg/odo/cli/component/create_helpers.go

```diff
@@ -91,3 +92,16 @@ func (co *CreateOptions) DevfileJSON() error {
 	machineoutput.OutputSuccess(cfd.GetComponent())
 	return nil
 }
+
+// GetComponentTypeFromDevfile returns component type from the devfile metadata
+func GetComponentTypeFromDevfile(metadata devfile.DevfileMetadata) string {
+	var componentType string
+	if metadata.ProjectType != "" {
+		componentType = metadata.ProjectType
+	} else if metadata.Language != "" {
+		componentType = metadata.Language
+	} else {
+		componentType = "Not available"
```

Comment:
Can we make  a variable for`Not available` in this file?

Reply:
Do you mean `const`?

Reply:
Yep